### PR TITLE
Add test for logging policy with custom date format

### PR DESCRIPTION
**Pull request by JaurbanRH**
Test for https://issues.redhat.com/browse/THREESCALE-6594

### CODE REVIEW

File: testsuite/tests/apicast/policy/logging/test_date_logging.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+
+def test_date_logging(api_client, staging_gateway):
+    """
+    Test that the logs contain the log in the correct date format
+    """
+    api_client().get("/anything")
+    logs = staging_gateway.get_logs()
+    today = datetime.now(timezone.utc).strftime("%m %d, %Y")
+    assert today in logs
```
Comment (pehala) [inline testsuite/tests/apicast/policy/logging/test_date_logging.py:33]: You should use self-managed APIcast as if this test was run two times in a single day the previous value would be the same and also present in the logs.

**Review by pehala — APPROVED** [review]

File: testsuite/tests/apicast/parameters/test_date_logging.py [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+pytestmark = [
+    pytest.mark.issue("https://issues.redhat.com/browse/THREESCALE-6594"),
+    pytest.mark.required_capabilities(Capability.LOGS),
+]
+
+
+@pytest.fixture(scope="module")
+def policy_settings():
+    """Customize the access logs format"""
```
Comment (mganisin) [inline testsuite/tests/apicast/parameters/test_date_logging.py:19]: `policy_settings` works nicely with predefined customized `service`. if that doesn't exist (to share simpler definition between other tests) neither `policy_settings` makes sense.

1. Either place policy definition directly to `service` fixture
2. or put the `service` fixture honoring `policy_settings` to conftest here or upstrairs

**Review by mganisin — CHANGES_REQUESTED** [review]

File: testsuite/tests/apicast/parameters/test_date_logging.py [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+    return rawobj.PolicyConfig("logging", {
+
```
Comment (mganisin) [inline testsuite/tests/apicast/parameters/test_date_logging.py:21]: Sorry, I don't understand line no.21 :-)

**Review by mdujava — DISMISSED** [review]

File: testsuite/tests/apicast/parameters/test_date_logging.py [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+from datetime import datetime, timezone
+
+import pytest
+
+from testsuite import rawobj
+from testsuite.capabilities import Capability
+
+pytestmark = [
+    pytest.mark.issue("https://issues.redhat.com/browse/THREESCALE-6594"),
+    pytest.mark.required_capabilities(Capability.LOGS),
```
Comment (pehala) [inline testsuite/tests/apicast/parameters/test_date_logging.py:13]: I am missing skipif for version

**Review by mganisin — APPROVED** [review]

**Review by pehala — APPROVED** [review]

**Pull request merged**